botocore/httpsession.py

```diff
@@ -19,8 +24,12 @@
     PROTOCOL_TLS,
     OP_NO_SSLv2,
     OP_NO_SSLv3,
+    is_ipaddress,
     ssl,
 )
+from urllib3.util.url import parse_url
+
+from botocore import utils
```

Comment:
We merged #2660 to fix this. Let's change the path to `from botocore.compat import IPV6_ADDRZ_RE` which will handle this. I don't think we'll want to add a generic `utils` namespace.

Reply:
Ok, I didn't realize that. let me revert that back.